bigquery/storage/managedwriter/connection.go

```diff
@@ -148,6 +298,68 @@ func (co *connection) getStream(arc *storagepb.BigQueryWrite_AppendRowsClient, f
 	return co.arc, co.pending, co.err
 }
 
+// lockingAppend handles a single append request on a connection.
+func (c *connection) lockingAppend(pw *pendingWrite) error {
```

Comment:
are you planning to remove the `lockingAppend` method on both `connection` and `ManagedStream` on this PR or you are going to remove from `ManagedStream` already ?

Reply:
When I reach the refactor point where I modify ManagedStream most of the functionality goes away, lockingAppend and other connection-oriented funcs will be removed.